### PR TITLE
Add new CPUs from RAL

### PR DESCRIPTION
**Pull request by jrha**

### CODE REVIEW

**Review by Pansanel — COMMENTED** [review]
The chipset attribute is not set in all files. As it seems to be redundant with the type entry, is it really necessary?

File: hardware/cpu/intel/xeon_gold_5218.pan [hunk 1]
```diff
@@ -0,0 +1,10 @@
+structure template hardware/cpu/intel/xeon_gold_5218;
+
+"manufacturer" = "Intel";
+"model" = "Intel(R) Xeon(R) Gold 5120 CPU @ 2.20 GHz";
```
Comment (Pansanel) [inline hardware/cpu/intel/xeon_gold_5218.pan:4]: The model name does not fit with the template name.

Comment (jrha) [reply hardware/cpu/intel/xeon_gold_5218.pan]: Well spotted, thanks!

**Comment by jrha** [comment]
@Pansanel thanks, I've removed the chipset attribute and fixed the `xeon_gold_5218`.

**Review by Pansanel — APPROVED** [review]

**Pull request merged**